api/api.go

```diff
@@ -112,19 +102,17 @@ type PoolPutRequest struct {
 }
 
 type BranchPostRequest struct {
-	Name     string `json:"name"`
-	ParentID string `json:"parent_id"`
-	At       string `json:"at"`
+	Name   string `json:"name"`
+	Commit string `json:"commit"`
 }
 
 type BranchMergeRequest struct {
 	At string `json:"at"`
 }
 
-type CommitRequest struct {
-	Author  string  `zng:"author"`
-	Date    nano.Ts `zng:"date"`
-	Message string  `zng:"message"`
+type CommitMessage struct {
```

Comment:
Nit: I think `Commit`, `CommitInfo`, or something like that would be a better name for this.

Reply:
But it's a message.  I was going to change the field name from Message to Body.  So it's a message with an author and a body.  Unless it changes to contain broader and less related information, I wouldn't call it generically "info".

Reply:
And "Commit" by itself implies to me an operation not an annotation.

Reply:
The reason `CommitMessage` feels wrong to me is that it doesn't match the model in `lake/actions.Commit` (or Git or Mercurial), in which author and message are distinct attributes of the commit object (rather than author being an attribute of the message).

Reply:
Ok, how about we fix it when we do #3001 as it will have a different look and feel then...

Reply:
👍